4_advanced_usage/30_technique_creation.txt

```diff
@@ -5,7 +5,94 @@ configuration and system administration needs. You can also create your own
 Techniques, to implement new functionality or configure new services. This
 paragraph will walk you through this process.
 
-==== Prequisites
+==== Use ncf-builder
+
+Easiest way to create your own Techniques is to use ncf-builder,
+a web interface to create and manage Technique based on ncf.
+
+Creating a technique in ncf-builder will generate a Technique for Rudder automatically
+that you can use as a Directive to apply on your Nodes.
+
+For more information about ncf and ncf-builder, you can visit: http://www.ncf.io/
+
+
+===== Organisation
```

Comment:
I propose to change the organisation of the following chapter so that they are more cleanly splitted between "things a simple user may be interested on" and "things a advanced user want to know to debug":

===== Logs [in place of Error messages]

....

===== Understanding how Technique Editor works

In this chapter, we are giving an overwiew about how the Technique Editor works and how it is integrated with the main Rudder application. 

===== Directory layout [in place of organisation]
....

===== Technique Editor integration with Rudder

...

====== Hooks

[missing one intro sentence: one each POST request, pre- and post- hooks are executed by Technique Editor. This hooks are used for the Rudder integration to help transform pure ncf Technique into Rudder one.]

....